### ISSUE TITLE
Update EOP IP address ranges

### ISSUE THREAD

**Issue by stanvx** [MicrosoftDocs/OfficeDocs-o365seccomp#383]
The EOP list has now been updated within the REST API URL - https://endpoints.office.com/endpoints/worldwide?clientrequestid=b10c5ed1-bad1-445f-b386-b919946339a7

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 90bba8f1-54aa-85f9-a311-2b4240c40202
* Version Independent ID: ae5462b5-141b-cef2-72dc-3f62f19b708c
* Content: [Exchange Online Protection IP addresses](https://docs.microsoft.com/en-us/office365/securitycompliance/eop/exchange-online-protection-ip-addresses#feedback)
* Content Source: [SecurityCompliance/eop/exchange-online-protection-ip-addresses.md](https://github.com/microsoftdocs/officedocs-o365seccomp/blob/public/SecurityCompliance/eop/exchange-online-protection-ip-addresses.md)
* Service: **o365-seccomp**
* GitHub Login: @JoeDavies-MSFT
* Microsoft Alias: **josephd**

**Comment by denisebmsft** [comment]
@stanvx thank you for the feedback. We are looking into this, and an update will be posted here soon.

**Comment by shivtorov** [comment]
List of IP subnets here does not match the list in Table row #10 at https://docs.microsoft.com/en-au/office365/enterprise/urls-and-ip-address-ranges#exchange-online

Which list is more accurate/up-to-date?

**Comment by AndyLiggettEXP** [comment]
Same problem here. Moreover I have authentications coming from EOP IP addresses that arent listed on the JSON which I am currently denying.

**Comment by AndyLiggettEXP** [comment]
Here's the response I got from an MS ticket:

"We confirmed from our team that both the mentioned articles are recommended for the working of EOP and Office 365, there are a few IP addresses range which might not be listed in Office 365 URLs and IP ranges, however are listed in EOP IP list hence IPs and URLs from both the articles must be whitelisted at the firewall to avoid the connectivity issues."


So in short the endpoints JSON file is not definitive and therefore effectively useless, and no comment on why I see *authentications* coming from EOP addresses.

Not that helpful.

Cheers

**Comment by stukey** [comment]
I would like a definite answer on this topic, too. We also opened a case with Microsoft and were told to rely on the Exchange Online IP address list provided by the REST API, not on this page. But the previous customer was advised to use both sources. So which is the correct answer?

**Comment by scanum** [comment]
@officedocsbot assign @scanum

**Comment by JoeDavies-MSFT** [comment]
@stanvx @shivtorov @AndyLiggettEXP @stukey The current and only list of IP address ranges for Exchange Online is here: https://docs.microsoft.com/en-us/office365/enterprise/urls-and-ip-address-ranges#exchange-online

The previous article has been removed.

**Comment by AndyLiggettEXP** [comment]
Were the IPs from the EOP article added to the endpoint list?
